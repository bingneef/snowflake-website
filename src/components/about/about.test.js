import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";

import About from "./about";

describe("About", () => {
  it("renders without error", () => {
    const div = document.createElement("div");
    ReactDOM.render(<About />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it("matches the snapshot", () => {
    const component = renderer.create(<About title="About" />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
