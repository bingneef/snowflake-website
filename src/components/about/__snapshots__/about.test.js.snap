// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About matches the snapshot 1`] = `
<section
  className="hero is-info is-fullheight"
  id="about"
>
  <div
    className="hero-body"
  >
    <div
      className="container"
    >
      <h1
        className="title is-1 has-text-centered title"
      >
        Whats all this about?
      </h1>
      <div
        className="columns"
      >
        <div
          className="column"
        >
          <h1
            className="title title"
          >
            Who adds the fluffy?
          </h1>
          <p>
            Hello! I am an full-stack developer from Noordwijk, Netherlands.
          </p>
          <br />
          <p>
            So, if you have a new and awesome project, get in touch! And if your project is not 
            <span
              className="underline"
            >
              that
            </span>
             
            awesome, challenge me to make it awesome!
          </p>
        </div>
        <div
          className="column"
        >
          <h1
            className="title title"
          >
            Everyone needs skills!
          </h1>
          <div
            className="dataLine"
          >
            <p>
              <strong
                className="data-line-content"
              >
                <svg
                  aria-hidden="true"
                  className="svg-inline--fa fa-magic fa-w-16 "
                  data-icon="magic"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  style={Object {}}
                  viewBox="0 0 512 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M224 96l16-32 32-16-32-16-16-32-16 32-32 16 32 16 16 32zM80 160l26.66-53.33L160 80l-53.34-26.67L80 0 53.34 53.33 0 80l53.34 26.67L80 160zm352 128l-26.66 53.33L352 368l53.34 26.67L432 448l26.66-53.33L512 368l-53.34-26.67L432 288zm70.62-193.77L417.77 9.38C411.53 3.12 403.34 0 395.15 0c-8.19 0-16.38 3.12-22.63 9.38L9.38 372.52c-12.5 12.5-12.5 32.76 0 45.25l84.85 84.85c6.25 6.25 14.44 9.37 22.62 9.37 8.19 0 16.38-3.12 22.63-9.37l363.14-363.15c12.5-12.48 12.5-32.75 0-45.24zM359.45 203.46l-50.91-50.91 86.6-86.6 50.91 50.91-86.6 86.6z"
                    fill="currentColor"
                    style={Object {}}
                  />
                </svg>
                 CREATING MAGIC
              </strong>
            </p>
            <p>
              React, React-native, NodeJS, Ruby on Rails, VueJS
            </p>
          </div>
          <div
            className="separator"
          />
          <div
            className="dataLine"
          >
            <p>
              <strong
                className="data-line-content"
              >
                <svg
                  aria-hidden="true"
                  className="svg-inline--fa fa-bed fa-w-20 "
                  data-icon="bed"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  style={Object {}}
                  viewBox="0 0 640 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M176 256c44.11 0 80-35.89 80-80s-35.89-80-80-80-80 35.89-80 80 35.89 80 80 80zm352-128H304c-8.84 0-16 7.16-16 16v144H64V80c0-8.84-7.16-16-16-16H16C7.16 64 0 71.16 0 80v352c0 8.84 7.16 16 16 16h32c8.84 0 16-7.16 16-16v-48h512v48c0 8.84 7.16 16 16 16h32c8.84 0 16-7.16 16-16V240c0-61.86-50.14-112-112-112z"
                    fill="currentColor"
                    style={Object {}}
                  />
                </svg>
                 STUFF I CAN DREAM
              </strong>
            </p>
            <p>
              Typescript, Jest, GatsbyJS, MySQL, MongoDB, Parcel/Webpack, SCSS
            </p>
          </div>
          <div
            className="separator"
          />
          <div
            className="dataLine"
          >
            <p>
              <strong
                className="data-line-content"
              >
                <svg
                  aria-hidden="true"
                  className="svg-inline--fa fa-bug fa-w-16 "
                  data-icon="bug"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  style={Object {}}
                  viewBox="0 0 512 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M511.988 288.9c-.478 17.43-15.217 31.1-32.653 31.1H424v16c0 21.864-4.882 42.584-13.6 61.145l60.228 60.228c12.496 12.497 12.496 32.758 0 45.255-12.498 12.497-32.759 12.496-45.256 0l-54.736-54.736C345.886 467.965 314.351 480 280 480V236c0-6.627-5.373-12-12-12h-24c-6.627 0-12 5.373-12 12v244c-34.351 0-65.886-12.035-90.636-32.108l-54.736 54.736c-12.498 12.497-32.759 12.496-45.256 0-12.496-12.497-12.496-32.758 0-45.255l60.228-60.228C92.882 378.584 88 357.864 88 336v-16H32.666C15.23 320 .491 306.33.013 288.9-.484 270.816 14.028 256 32 256h56v-58.745l-46.628-46.628c-12.496-12.497-12.496-32.758 0-45.255 12.498-12.497 32.758-12.497 45.256 0L141.255 160h229.489l54.627-54.627c12.498-12.497 32.758-12.497 45.256 0 12.496 12.497 12.496 32.758 0 45.255L424 197.255V256h56c17.972 0 32.484 14.816 31.988 32.9zM257 0c-61.856 0-112 50.144-112 112h224C369 50.144 318.856 0 257 0z"
                    fill="currentColor"
                    style={Object {}}
                  />
                </svg>
                 WITH SOME BUGS
              </strong>
            </p>
            <p>
              PHP, Android
            </p>
          </div>
          <div
            className="separator"
          />
          <div
            className="dataLine"
          >
            <p>
              <strong
                className="data-line-content"
              >
                <svg
                  aria-hidden="true"
                  className="svg-inline--fa fa-terminal fa-w-20 "
                  data-icon="terminal"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  style={Object {}}
                  viewBox="0 0 640 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M257.981 272.971L63.638 467.314c-9.373 9.373-24.569 9.373-33.941 0L7.029 444.647c-9.357-9.357-9.375-24.522-.04-33.901L161.011 256 6.99 101.255c-9.335-9.379-9.317-24.544.04-33.901l22.667-22.667c9.373-9.373 24.569-9.373 33.941 0L257.981 239.03c9.373 9.372 9.373 24.568 0 33.941zM640 456v-32c0-13.255-10.745-24-24-24H312c-13.255 0-24 10.745-24 24v32c0 13.255 10.745 24 24 24h304c13.255 0 24-10.745 24-24z"
                    fill="currentColor"
                    style={Object {}}
                  />
                </svg>
                 CURRENTLY LEARNING
              </strong>
            </p>
            <p>
              NextJS
            </p>
          </div>
        </div>
        <div
          className="column personal-details"
        >
          <h1
            className="title title"
          >
            ome personal details.
          </h1>
          <div
            className="columns is-mobile"
          >
            <div
              className="column is-narrow data-line"
            >
              <p>
                <strong>
                  Name
                </strong>
              </p>
              <p>
                <strong>
                  Age
                </strong>
              </p>
              <p>
                <strong>
                  Email
                </strong>
              </p>
            </div>
            <div
              className="column data-line"
            >
              <p>
                Bing Steup
              </p>
              <p>
                30
                 Years
              </p>
              <p>
                <a
                  className="isLink"
                  href="mailto:bingsteup@gmail.com"
                >
                  bingsteup@gmail.com
                </a>
              </p>
            </div>
          </div>
          <div
            className="socialIcons"
          >
            <a
              href="https://www.github.com/bingneef"
              target="_blank"
            >
              <img
                src="/static/assets/images/social/github.svg"
              />
            </a>
            <a
              href="https://www.instagram.com/bingneef"
              target="_blank"
            >
              <img
                src="/static/assets/images/social/instagram.svg"
              />
            </a>
            <a
              href="https://www.flickr.com/photos/bingneef"
              target="_blank"
            >
              <img
                src="/static/assets/images/social/flickr.svg"
              />
            </a>
            <a
              href="https://www.linkedin.com/in/bing-steup-6595a6b1?trk=hp-identity-name"
              target="_blank"
            >
              <img
                src="/static/assets/images/social/linkedin.svg"
              />
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
</section>
`;
